supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc

```diff
@@ -82,6 +82,17 @@ Use the hyphenated "fault-tolerant" when using it as an adjective.
 
 *See also*:
 
+//Add FedRAMP terminology Issue #395
+[[fedramp]]
+==== image:images/yes.png[yes] FedRAMP® (noun)
+*Description*: The _Federal Risk and Authorization Management Program (FedRAMP®)_ is managed by the FedRAMP Program Management Office. Add the registration symbol (®) on first use in a document. For more information, see the link:https://www.fedramp.gov/assets/resources/documents/FedRAMP_Branding_Guidance.pdf[FedRAMP Brand Guide].
```

Comment:
I can't find guidance anywhere on this, but should we be avoiding the actual TM and R symbols in the Asciidoc code and using (TM) and (R) instead?

Reply:
Hi @bburt-rh . I added guidance to the[ OpenShift guidance doc](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#third-party-vendor-product-names). Will this help here? From experience with pltform rendering, using (R) proved the safest option.

Reply:
Thanks, @dfitzmau. I also found a[ section in the IBM Style guide](https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting#contents) where they say to "Always use the semantic tagging of your authoring tool to produce the correct output." So I think this PR needs to remove the symbols from the code.

Reply:
IBM style guide says w/r/t its own (R) and (TM): "The first reference to an IBM product name in each topic or book must be preceded by "IBM" and must include the designated trademark symbols ® or ™ as appropriate. "

Also,https://www.ibm.com/docs/en/ibm-style?topic=punctuation-symbols#registered_trademark_symbol

Reply:
We already have many other occurrences of the ® symbol in the SSG, so I think the current formatting is fine. If we want to change it, we should change all occurrences in a new PR.